WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -75,6 +75,8 @@ extension ProductFormViewController: UITableViewDelegate {
         case .primaryFields(let rows):
             let row = rows[indexPath.row]
             switch row {
+            case .name:
+                editProductName()
             case .description:
                 editProductDescription()
             default:
```

Comment:
I get a warning here. Default will never be executed. Since all cases are accounted for, you can remove the default case :)